### PR TITLE
19841: defaultHostName variable should work with bootstrap.properties

### LINKED ISSUE #19841: defautHostName does not get picked up from bootstrap.properties for cfw
**Describe the bug**  
In 21.0.0.8 and earlier, the Liberty code would not replace the value from bootstrap.properties with the variable definition from defaultInstances.  That issue was fixed on 21.0.0.8.

That exposed an issue in Channel Framework where defaultHostName is set to a value "localhost" by default and is not being allowed to be overwritten if using the same name.

**Steps to Reproduce**  
In bootstrap.properties define the config variable, defaultHostName as follows; 
`defaultHostName=anyhostname`

In server.xml use this property:
```
<httpEndpoint id="defaultHttpEndpoint"

   host="${defaultHostName}"/>
```

The hostname will not be picked up properly and will be assigned the locahost.

**Expected behavior**  
The defaultHostName defined in bootstrap.properties should be picked up in the httpEndpoint definition of server.xml.

**Diagnostic information:**  
 - OpenLiberty Version: 21.0.0.9 - 22.0.0.2
 - Affected feature(s): no specific feature 
 - Java Version: n/a
 - server.xml configuration : httpEndpoints in server.xml which use a configuration variable defined in bootstrap.properties

**Additional context**

### PR DESCRIPTION
**Pull request by loriadi**
fixes #19841

### CODE REVIEW

**Comment by loriadi** [comment]
fixes #19841

**Comment by loriadi** [comment]
#build

**Comment by loriadi** [comment]
#build

**Comment by loriadi** [comment]
After the failing test was disabled in closed liberty, all tests now pass with this change.

File: dev/com.ibm.ws.channelfw/resources/OSGI-INF/wlp/defaultInstances.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2017 IBM Corporation and others.
+    Copyright (c) 2017,2022 IBM Corporation and others.
```
Comment (pnicolucci) [inline dev/com.ibm.ws.channelfw/resources/OSGI-INF/wlp/defaultInstances.xml]: small comment here, please add a space after the `,`. Thanks!

**Comment by loriadi** [comment]
#build

**Review by pnicolucci — APPROVED** [review]

**Comment by loriadi** [comment]
#run-libby-bot

**Pull request merged**